notebooks/colab_finetuning.ipynb

```diff
@@ -0,0 +1,2022 @@
+        "!wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ./miniconda.sh\n",
+        "!chmod +x miniconda.sh\n",
+        "!bash ./miniconda.sh -b -u -p /usr/local\n",
+        "!conda env update -q -n base -f ./environment.yml"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "432eebaf-00b8-42bf-b927-fd651e6ab94d",
+      "metadata": {
+        "ExecuteTime": {
+          "end_time": "2024-02-16T22:51:36.573559Z",
+          "start_time": "2024-02-16T22:51:36.569490Z"
+        },
+        "id": "432eebaf-00b8-42bf-b927-fd651e6ab94d",
+        "jupyter": {
+          "outputs_hidden": false
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%load_ext autoreload\n",
+        "%autoreload 2"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "c566507e-1012-4415-82ba-7498950e0b6c",
+      "metadata": {
+        "id": "c566507e-1012-4415-82ba-7498950e0b6c"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "import sys\n",
+        "\n",
+        "sys.path.append('/usr/local/lib/python3.9/site-packages')\n",
+        "# define the name of the project root directory\n",
+        "project_root_dir_name = \"metl\"\n",
+        "\n",
+        "# find the project root by checking each parent directory\n",
+        "current_dir = os.getcwd()\n",
+        "while os.path.basename(current_dir) != project_root_dir_name and current_dir != os.path.dirname(current_dir):\n",
+        "    current_dir = os.path.dirname(current_dir)\n",
+        "\n",
+        "# change the current working directory to the project root directory\n",
+        "if os.path.basename(current_dir) == project_root_dir_name:\n",
+        "    os.chdir(current_dir)\n",
+        "else:\n",
+        "    print(\"project root directory not found\")\n",
+        "\n",
+        "# add the project code folder to the system path so imports work\n",
+        "module_path = os.path.abspath(\"code\")\n",
+        "if module_path not in sys.path:\n",
+        "    sys.path.append(module_path)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "19876208-66f9-46b5-8f50-8e798fa815a4",
+      "metadata": {
+        "id": "19876208-66f9-46b5-8f50-8e798fa815a4"
+      },
+      "source": [
+        "# Acquire an experimental dataset\n",
+        "\n",
+        "For demonstration purposes, this repository contains the [avGFP dataset](https://github.com/gitter-lab/metl/tree/main/data/dms_data/avgfp) from [Sarkisyan et al. (2016)](https://doi.org/10.1038/nature17995).\n",
+        "See the [metl-pub](https://github.com/gitter-lab/metl-pub) repository to access the other experimental datasets we used in our preprint.\n",
+        "See the README in the [dms_data](https://github.com/gitter-lab/metl/tree/main/data/dms_data/avgfp/data/dms_data) directory for information about how to use your own experimental dataset."
```

Comment:
This global link and some others are broken. Some are still relative. Please double check them all.

Reply:
@agitter The broken links were links to python or ipynb files that are broken in the original notebooks as well.

Additionally, the broken global links are actually linking to pages that no longer exist. The only way to get to those files are by going to older commits, when they did exist, like the other notebook does. 

I do not know what you want to do with that data. I fixed the ipynb links, but I will wait on the broken globals because those files are no longer in the current version of the main repository. 

An example of that is this link here for the dms_data that is in the notebook:  https://github.com/gitter-lab/metl/tree/main/data/dms_data/avgfp/data/dms_data

But the actual file was deleted though can be found in an earlier commit:  https://github.com/gitter-lab/metl/tree/45237bbc4c6cfae1d4657930bdb732ec41a3fc79/data/dms_data

Are we supported this still? Does this need to be referenced or talk about? And if it does, why is it not included in the current version of main? 

I'm confused on these so I am leaving them, again, I fixed the broken links to python files and notebooks (that were also broken in the normal finetuning notebook)

Reply:
Got it. This is a question for @samgelman.

Reply:
In this case, the link https://github.com/gitter-lab/metl/tree/main/data/dms_data/avgfp/data/dms_data is incorrect, it should just be https://github.com/gitter-lab/metl/tree/main/data/dms_data/avgfp.